modules/nf-core/bedtools/bamtobed/tests/main.nf.test.snap

```diff
@@ -0,0 +1,31 @@
+{
+    "sarscov2 - bam": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test"
+                        },
+                        "test.bed:md5,f7d05b790b2c1c8e598d52828e7f81b7"
```

Comment:
:( for some reason this is different from the pytest md5 at https://github.com/nf-core/modules/pull/4461/files#diff-e62d1fcba9d0b9716fecb3e1561ef0717abb3c8c684a1c230fb5cc1dfb519ea7L8

Reply:
found it! copy paste error - need to use test_single_end_bam, not test_paired_end_bam